src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "name": "Google BigQuery API",
+    "documentation": "https://cloud.google.com/bigquery/docs/reference/rest/v2/",
```

Comment:
That's a landing page for BigQuery, not for the BigQuery java API client library.